dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -504,6 +504,7 @@
 				@(_MonoLibrary -> 'MonoLibrary=%(Identity)')
 				MtouchFloat32=$(MtouchFloat32)
 				Optimize=$(_BundlerOptimize)
+				OptimizeClassHandles=$(OptimizeClassHandles)
```

Comment:
You already added an optimization option here:

https://github.com/xamarin/xamarin-macios/commit/62634bdc7a783c54b7117b36920d2e65f5458b41

can't that one be used instead of adding another one?